forcedotcom/salesforcedx-vscode
# feat: add E4D to the expanded pack

### What does this PR do?
- Add E4D to the expanded pack 
- ~Update the expanded pack ReadME to include E4D~ Handled by DocWriter PR 

### What issues does this PR fix or reference?
@W-15734847@

### Functionality Before
No E4D in expanded pack

### Functionality After
E4D expanding our horizons

### Testing Notes
- Verify from clean install (no SFVCE installed) installing the expanded pack gets everything activated including E4D
- Update from an install expanded pack -> EP w/ E4D vsix and verify everything loads successfully.
- Install both the standard and expanded pack w/ E4D and ensure everything starts up (like 1/2 our users run this way :shrug:) 
- Uninstall expanded pack and verify E4D gets uninstalled.
